components/pipeline-service/development/kustomization.yaml

```diff
@@ -8,8 +8,8 @@ commonAnnotations:
   argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
 
 resources:
-  - git::https://github.com/openshift-pipelines/pipeline-service.git/developer/openshift/gitops/argocd/pipeline-service?ref=062d00d304d6fba316085784f3b948217fc99d12
-  - git::https://github.com/openshift-pipelines/pipeline-service.git/developer/openshift/gitops/argocd/pipeline-service-storage?ref=062d00d304d6fba316085784f3b948217fc99d12
+  - git::https://github.com/openshift-pipelines/pipeline-service.git/developer/openshift/gitops/argocd/pipeline-service?ref=14431236de0f1e3d3a9e27c145197d1a45942794
+  - git::https://github.com/openshift-pipelines/pipeline-service.git/developer/openshift/gitops/argocd/pipeline-service-storage?ref=14431236de0f1e3d3a9e27c145197d1a45942794
```

Comment:
@Roming22 This is not going to work since pipeline-service-storage is not available anymore in pipeline-service repo it was replaced by two apps: minio + postgres.

I have sent this PR https://github.com/openshift-pipelines/pipeline-service/pull/621 to remediate to that.

Reply:
Version updated to include the fix.